.travis.yml

```diff
@@ -3,10 +3,18 @@ python:
   - 3.4
 addons:
   apt:
+    sources:
+      - ubuntu-toolchain-r-test
```

Comment:
Ahh gotcha. The only reason I'm currently using the -test ppa is because it is the only one in Travis' [whitelist](https://github.com/travis-ci/apt-source-whitelist/blob/master/ubuntu.json#L208) and hence available through the apt addon. It appears that the reason for this is the regular ppa does not include 4.8+ for Ubuntu 12.04. While it's in the published list, you can see that the list of packages *actually* available for 12.04 in the [ppa repo](http://ppa.launchpad.net/ubuntu-toolchain-r/ppa/ubuntu/dists/precise/main/binary-amd64/Packages) are... none... but the [test repo](http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu/dists/precise/main/binary-amd64/Packages) contains what we want.

This doesn't answer if it's as reliable as we would like, but it's the only solution I could find (aside form compiling from source on Travis). We should check back periodically to see if we can switch to the release branch.